Homework/Homework/Program.cs

```diff
@@ -88,4 +100,18 @@ public static int MiddleOf(int a, int b, int c)
         }
         return a;
     }
+    public static string RemoveStartSpaces(string text)
+    {
+        int index = 0;
+        while (char.IsWhiteSpace(text[index]))
+        {
+            index++;
+            if (index == text.Length)
+            {
+                return null;
```

Comment:
```suggestion
                return string.Empty;
```
Не всегда стоит возвращать null, потому что если вызывать твой метод снаружи может быть опасно из за ошибки null reference, если ты пометил что тип возвращаемого значения строка, лучше возвращай пустую строку